examples/resources/spacelift_scheduled_delete_stack/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "spacelift_stack" "k8s-core" {
+  // ...
+}
+
+// at a given timestamp (unix)
+resource "spacelift_scheduled_task" "k8s-core-delete" {
+  stack_id = spacelift_stack.k8s-core.id
+
+  at               = "1663336895"
```

Comment:
Is it worth supporting ISO-8601 to be a bit more user-friendly? So `at = "2022-09-29T10:00:00Z"` for example. Maybe we could support both?

Reply:
this would require a bit more work in the API, we only support parsing UNIX timestamps. I can add it to the V1 launch suggestions.

Reply:
I figured we could just do it in the provider, but it's a nice to have anyway.